test-files/cmd/glacial/serial/RndServiceNodeDensityTilesCmdTest.sh

```diff
@@ -4,6 +4,7 @@ set -e
 OUTPUT_DIR=test-output/cmd/glacial/RndServiceNodeDensityTilesCmdTest
 GOLD_FILES_DIR=test-files/cmd/glacial/RndNodeDensityTilesCmdTest
 mkdir -p $OUTPUT_DIR 
+mkdir -p $HOOT_HOME/tmp
```

Comment:
This unit test relied on the `$HOOT_HOME/tmp` directory to exist but never created it.  The ordering of tests changed so this was run before another test that created the directory.  Now this is self-sufficient.

Reply:
cool, thank.  That was killing why this test was failing all of the sudden.